pkg/esp8266_sdk/bootloader/sdkconfig.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2021 iosabi
```

Comment:
It doesn't make any difference for the purpose of copyright protection. Also having individuals listed at the top of each file is not very accurate since the copyright applies to the specific contribution: if I add a function to somebody else's copyrighted file I hold the copyright for that function and they hold the copyright for the rest. When suggestions are made in code reviews in the form of patches that's even more debatable. Many projects just list copyright "the [project] authors" to avoid all this anyway since it doesn't make any difference. I'd prefer to leave it like this.

Reply:
Unfortunately, I don't know what the policy of the project owners is. At least there is a statement in the [coding conventions](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md) there is a statement:
```
* All files contain the copyright note and the author.
```
whatever author means in this context. I haven't seen authors use just a synonym or their GitHub user yet. Maybe @kaspar030 as a project owner can clarify.

Reply:
@kaspar030 ping

Reply:
Maybe @tcschmidt as a project owner can clarify the policy on using the author's name in copyright notices for this project? I would like to move forward with this PR.

Reply:
No further concerns.